zboxcore/sdk/sdk.go

```diff
@@ -888,6 +893,60 @@ func CreateAllocationForOwner(name string, owner, ownerpublickey string,
 	return
 }
 
+func getAllocationBlobbers(owner, ownerpublickey string,
+	datashards, parityshards int, size, expiry int64,
+	readPrice, writePrice PriceRange, mcct time.Duration) ([]string, error) {
+
+	var allocationRequest = map[string]interface{}{
+		"data_shards":                   datashards,
+		"parity_shards":                 parityshards,
+		"size":                          size,
+		"owner_id":                      owner,
+		"owner_public_key":              ownerpublickey,
+		"expiration_date":               expiry,
+		"read_price_range":              readPrice,
+		"write_price_range":             writePrice,
+		"max_challenge_completion_time": mcct,
+	}
+
+	allocationData, _ := json.Marshal(allocationRequest)
+
+	params := make(map[string]string)
+	params["allocation_data"] = string(allocationData)
+
+	allocBlobber, err := zboxutil.MakeSCRestAPICall(STORAGE_SCADDRESS, "/get_alloc_blobbers", params, nil)
```

Comment:
this endpoint will need some pagination as well, as will be directly called from clients

Reply:
It has pagination somehow, response is limited with `MaxBlobbersPerAllocation` so it will never return too much ids as `MaxBlobbersPerAllocation` won't be big enough to make some problems for us